### PR TITLE
Updates to code/data pre-2021 FACT workshop.

### PR DESCRIPTION
**Pull request by jackVanish**
Updates to code and data in advance of the FACT 2020 December workshop.

### CODE REVIEW

File: code/day2/1_GlatosCode_FACT.R [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+coa %>% group_by(Tag.ID) %>% summarize(n())
+# Plot a COA
+# Get the data for only 'TQCS-1049273-2008-02-28'
+coa_single <- coa %>% filter(Tag.ID == 'TQCS-1049273-2008-02-28')
+view(coa_single)
+
+# We'll use raster to get the polygon
+library(raster)
+USA <- getData('GADM', country="USA", level=1)
+MD <- USA[USA$NAME_1=="Maryland",]
```
Comment (jdpye) [inline code/day2/1_GlatosCode_FACT.R:221]: This needs changing

**Review by jdpye — CHANGES_REQUESTED** [review]
Couple hanging references to the maryland shapefile from ACT, and some references to 'the bay we zoomed into earlier' without knowing whether we have actually done so. We may want to clean these comments up and make them less inter-referential generally, in case we are not taking all the modules we did when we first wrote them into a new workshop.

**Comment by jackVanish** [comment]
Corrected reference to MD and changed 'that bay we zoomed in on' to 'the area on which we zoomed in' in both FACT and ACT versions.

**Review by jdpye — APPROVED** [review]

**Comment by jdpye** [comment]
excellent. Approved and merged :D

**Pull request merged**